src/routes/docs-examples/extend/clipboard-manager/clipboardManagerProps.ts

```
@@ -0,0 +1,104 @@
+export let clipboardManagerProps = [
```

ESLint

[error] 'clipboardManagerProps' is never reassigned. Use 'const' instead.

(prefer-const)